webapp/components/topbar/SwitchPageButton.js

```diff
@@ -24,15 +23,21 @@ const styles = ({
   icon: { color: palette.common.white },
 });
 
+const editOptions = ["Create network", "Create and simulate", "Simulate"];
+const exploreOptions = ["Simulate", "Back to edit"];
+
 class SwitchPageButton extends Component {
   constructor (props) {
     super(props);
     this.handleClick = this.handleClick.bind(this);
   }
 
-  handleClick = (event) => {
+  handleClick = (selectedOption) => {
     const instantiate = this.props.automaticInstantiation || this.props.modelState === MODEL_STATE.NOT_INSTANTIATED;
-    if (!this.props.editModelPage) {
+    if(selectedOption === editOptions[2]){
```

Comment:
watch out with formatting, space required before and after parenthesis

Reply:
Instead of array check extract the options to constants and use them here for comparison.

```javascript
const CREATE_NETWORK = "Create network"
....
if (selectedOption == CREATE_NETWORK)
```

This way changing the order of the options array won't require changing the logic here and it's clear what selectedOption is in this moment.

Reply:
logic doesn't work for `Create network` and `Create and simulate network`, currently nothing happens, adapt and test all actions

Reply:
That's a really good point for decoupling the logic and the order of the menu items.